src/components/activity-page/text-box/text-box.scss

```diff
@@ -2,8 +2,12 @@
 
 .textbox {
   p {
-    margin: 0;
+    margin: 20px 0;
```

Comment:
20 on both sides seems big.  Won't back-to-back paragraphs get 40 pixels between them?  I would think that `10px 0` would be better (unless this is what the story requests).

Reply:
I take that back.  I think this looks fine (and I appear to be wrong about the duplicated margins - it doesn't become 40px):
![image](https://user-images.githubusercontent.com/5126913/99440610-aa907c80-28cb-11eb-884a-43141a74c60a.png)